Cargo.toml

```diff
@@ -11,3 +11,9 @@ repository = "https://github.com/lnicola/sd-notify"
 
 [package.metadata.docs.rs]
 targets = ["x86_64-unknown-linux-gnu"]
+
+[features]
+fdstore = ["dep:sendfd"]
+
+[dependencies]
+sendfd = { version = "0.4", optional = true }
```

Comment:
Nice, too bad it doesn't support `SCM_CREDENTIALS` for `sd_pid_notify` and friends.